tools/bqpipeline/example/example.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from bqpipeline.bqpipeline import BQPipeline
+import sys
+
+if __name__ == "__main__":
+    JOB_NAME = sys.argv[1]
+    PROJECT = sys.argv[2]
+    DATASET = sys.argv[3]
+    bq = BQPipeline(job_name=JOB_NAME,
+                    query_project=PROJECT,
+                    default_project=PROJECT,
+                    default_dataset=DATASET,
+                    json_credentials_path='credentials.json')
```

Comment:
You might want to explain to users how to create the file?

Reply:
Added credentials section to README.md